server/handlers/changefeed.go

```diff
@@ -0,0 +1,82 @@
+package handlers
+
+import (
+	"encoding/json"
+	"net/http"
+	"strconv"
+	"strings"
+
+	ctxu "github.com/docker/distribution/context"
```

Comment:
OK, then we might need to make another PR to do the cleanup, I've supposed it was like:
```
import (
        // std
        "os"

        // pre_1
        "github.com/..."

        // pre_2
        "golang.org/..."
)
```